src/View/Helper/Types/Type.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace BootstrapUI\View\Helper\Types;
+
+use ReflectionClass;
+use ReflectionException;
+
+/**
+ * Type
+ */
+abstract class Type implements TypeInterface
+{
+    /**
+     * Get all constant values of a class
+     *
+     * @return array|mixed
+     * @throws ReflectionException
```

Comment:
Should be `\ReflectionException` else IDEs and static analysers will consider it relative to current namespace.

Reply:
Even though its imported on the top of the file?
This is the way, PHPStorm proposed me to write it.